### DISCUSSION TITLE
Download do not match

### DISCUSSION THREAD

**Discussion by SaltyBill** [Shabinder/SpotiFlyer#2155] [Q&A]
Some downloaded songs is didn't match
Like i downloading Fake a smile
And it downloaded the remix of Fake a smile.
All i ask > All i ask ( cover)
All falls down > All falls down (Cover)
Alone pt.II > Alone pt.II (Color Music Choir)
I dunno why thats keeps happening:(

**Comment by kazi-shafiqul-alam** [comment]
I think the app doesn't download music directly from Spotify. It searches music across other music streaming platform like Jio Saavan, YouTube and many other. When it matches any title from those music streaming platforms. It just downloads from there. So, there it why there are song matching problems, and some songs even don't download because they are only available on Spotify. And every Spotify song downloader app or website does the same thing.

**Reply by ankushomg** [reply]
> I think the app doesn't download music directly from Spotify. It searches music across other music streaming platform like Jio Saavan, YouTube and many other. When it matches any title from those music streaming platforms. It just downloads from there. So, there it why there are song matching problems, and some songs even don't download because they are only available on Spotify. And every Spotify song downloader app or website does the same thing.

no it's download from that plateform,not from other plateform

**Reply by kazi-shafiqul-alam** [reply]
> > I think the app doesn't download music directly from Spotify. It searches music across other music streaming platform like Jio Saavan, YouTube and many other. When it matches any title from those music streaming platforms. It just downloads from there. So, there it why there are song matching problems, and some songs even don't download because they are only available on Spotify. And every Spotify song downloader app or website does the same thing.
> 
> no it's download from that plateform,not from other plateform


You really didn't check the log files. Then tell me why I cannot those songs which are only Spotify exclusive and not available on other platform? for example, rom$e$ - psycho. It is not available in any other platform beside Spotify. And I cannot download that song.
